packages/graphback-core/src/plugin/GraphbackCoreMetadata.ts

```diff
@@ -61,13 +62,13 @@ export class GraphbackCoreMetadata {
 
     /**
      * Helper for plugins to fetch all types that should be processed by Graphback plugins.
-     * To mark type as enabled for graphback generators we need to add `model` annotations over the type. 
-     * 
+     * To mark type as enabled for graphback generators we need to add `model` annotations over the type.
+     *
      * Returns all user types that have @model in description
-     * @param schema 
+     * @param schema
      */
     public getGraphQLTypesWithModel(): GraphQLObjectType[] {
-        const types = getModelTypesFromSchema(this.schema)
+        const types = getModelTypesFromSchema(this.schemaComposer.buildSchema())
```

Comment:
Is this currently stripping out the descriptions/markers? 
I think we need to figure out why descriptions/comments are missing when printing the schema

Reply:
Yes, it is. I shall open an issue at graphql-compose.

Reply:
Let's find what is causing this to know if that is something to do with merge that we are doing using GraphQL tools. if you create sample app that just use compose with comment does it also remove them?